projects/plugins/jetpack/extensions/index.json

```diff
@@ -28,6 +28,7 @@
 		"pinterest",
 		"podcast-player",
 		"premium-content",
+		"latex-formula",
```

Comment:
Let's start with only adding it a a beta block maybe, and switch to production after testing.

Reply:
Should it be `latex` or `latex-formula`?